src/utils.js

```diff
@@ -765,6 +785,7 @@ export const switchAffiliation = async (stripes, tenantId, move) => {
         perms: stripes.user.perms,
       },
     );
+    await handleUpdateUser(stripes);
```

Comment:
This function contains three `await`s. Would combining them into `Promise.all` give the same result?

Reply:
The `switchAffiliation` function will complete its execution faster. `Promise.all` allows multiple asynchronous operations to be executed concurrently. Async/await waits for the previous one to complete before continuing.

Reply:
Like this:
```
export const switchAffiliation = async (stripes, tenantId, move) => {
  if (stripes.okapi.tenant !== tenantId) {
    const updateTenantPromise = updateTenant(stripes.okapi, tenantId);
    
    const validateUserPromise = validateUser(
      stripes.okapi.url,
      stripes.store,
      tenantId,
      {
        token: stripes.okapi.token,
        user: stripes.user.user,
        perms: stripes.user.perms,
      },
    );
    
    const handleUpdateUserPromise = handleUpdateUser(stripes);
    
    await Promise.all([updateTenantPromise, validateUserPromise, handleUpdateUserPromise]);
    
    move();
  } else {
    move();
  }
 ```

Reply:
thanks, changed to Promise.all